### DISCUSSION TITLE
Welcome to react-google-maps-api Discussions!

### DISCUSSION THREAD

**Discussion by JustFly1984** [JustFly1984/react-google-maps-api#1860] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dakkapik** [comment]
Hello, first I would like to thank you guys for the work you have put into the project, I am currently using it on a personal project that uses a node js application and the google maps directions api to calculate the best route between a dynamic set of points in a city. Since am calculating this in the server side and getting the result there too. It would be great to just be able to diplay it using the directions render method on your project. But the object that my server returns and the one your directions service method returns are slightly different. my object seems to be missing a couple keys like request. I wasn't able to find how you are calling the api so I could replicate it.
If you could clarify how you are calling the directions api it would be of great help.
If you would be interested in taking a tour of my project i would be glad to show you around.
thanks for the help in advance.
